classes/omnect_uboot_embedded_version.bbclass

```diff
@@ -71,7 +71,9 @@
 #  - <version string> is the actual version, w/o terminating character
 #
 
-SRC_URI += "file://${DEPLOY_DIR_IMAGE}/omnect_bootloader_version"
+# we need the calculated bootloader version, so inherit class
+# omnect_bootloader_versioning alongside deploy
+inherit deploy omnect_bootloader_versioning
```

Comment:
it is computed either way. the removed line 74 will include the current computed version and the recipe will error if it isn't there yet.